modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/ServerWorker.java

```diff
@@ -107,110 +107,114 @@ public ServerWorker(final SourceRequest request,
     }
 
     public void run() {
-        if (log.isDebugEnabled()) {
-            log.debug("Starting a new Server Worker instance");
-        }
-        ConfigurationContext cfgCtx = sourceConfiguration.getConfigurationContext();        
-        msgContext.setProperty(Constants.Configuration.HTTP_METHOD, request.getMethod());
-
-        String method = request.getRequest() != null ?
-                request.getRequest().getRequestLine().getMethod().toUpperCase() : "";
-        
-        String uri = request.getUri();
-        String oriUri = uri;
-        
-        if (uri.contains(cfgCtx.getServicePath())) {
-            // discard up to servicePath
-            uri = uri.substring(uri.indexOf(cfgCtx.getServicePath()) +
-                    cfgCtx.getServicePath().length());
-            // discard [proxy] service name if any
-            int pos = uri.indexOf("/", 1);
-            if (pos > 0) {
-                uri = uri.substring(pos);
-            } else {
-                pos = uri.indexOf("?");
-                if (pos != -1) {
+        try {
```

Comment:
Please do not do code formatting in the entire class while sending fixes, it makes really hard to see the actual change you have done.

Reply:
Noted 👍